lib/Graph/Graph.cpp

```diff
@@ -1907,9 +1907,31 @@ void Function::verify() const {
     }
   }
 
+  std::unordered_map<const Variable *, const Node *> variablesWrittenTo;
```

Comment:
I thought about a set at first but decided to used a map to be able to print where another write happens.
Arguably, this is partially useful because we would only print two writes. I could either refactor the code to print all the writes or none and use a set.

What do you think?

Reply:
I do not see where you'd use the value from the map ;)

Reply:
ah, that's fine. I missed the error output from the varToFirstDef.